orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/NotifyDeployCompletedTask.java

```diff
@@ -40,13 +40,31 @@
 
   @Override
   public @Nonnull TaskResult executeInternal(
-      Stage stage,
-      MonitoredDeployStageData context,
-      DeploymentMonitorDefinition monitorDefinition) {
-    // TODO(mvulfson): actually populate the request data
+      Stage stage, DeploymentMonitorDefinition monitorDefinition) {
     DeploymentCompletedRequest request = new DeploymentCompletedRequest(stage);
-    monitorDefinition.getService().notifyCompleted(request);
 
+    request.setStatus(convertStageStatus((Boolean) stage.getContext().get("failure")));
```

Comment:
this (`....get("failure")`) should be called whatever the variable is called here:
https://github.com/spinnaker/orca/pull/3214/files#diff-e1c66b2ef51867abe920742e3093b21cR33
(e.g. `isFailure` as you have it now - unless you rename it as i suggested below)

Reply:
@marchello2000 I didn't really understand what you meant here ^. I've renamed the variable to 'hasDeploymentFailed' according to your suggestion.